From c73bb05ea456372b355ccdab700988507f21e474 Mon Sep 17 00:00:00 2001
From: Antonio Gomes <tonikitoo@igalia.com>
Date: Wed, 13 Sep 2023 22:22:17 +0000
Subject: [PATCH] Backport "[lacros] Include <string_view> from
 net/cert/pki/general_names.h"

This fixes a "no member named 'string_view' in namespace 'std'" build
error. The patch will be included in upstream's 119 release.

Upstream-Status: Backport [https://crrev.com/c/4862165]
Signed-off-by: Max Ihlenfeldt <max@igalia.com>
---
 net/cert/pki/general_names.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/cert/pki/general_names.h b/net/cert/pki/general_names.h
index e35e410..7704837 100644
--- a/net/cert/pki/general_names.h
+++ b/net/cert/pki/general_names.h
@@ -6,6 +6,7 @@
 #define NET_CERT_PKI_GENERAL_NAMES_H_

 #include <memory>
+#include <string_view>
 #include <vector>

 #include "net/base/net_export.h"
